.ci/travis/travis.sh

```diff
@@ -189,6 +189,8 @@ sonarqube)
   mvn -e clean package sonar:sonar \
        -Dsonar.host.url=https://sonarcloud.io \
        -Dsonar.login=$SONAR_TOKEN \
+       -Dsonar.projectKey=com.puppycrawl.tools:checkstyle \
+       -Dsonar.organization=checkstyle \
```

Comment:
Since these are parameters being sent to maven, won't this be the same if we added a `<sonar>` tag to the POM with all these values filled in, minus login of course. That would allow us to remove them here and have them in the POM. projectKey and organization could then be replaced with maven variables `${groupId}:${artifactId}` and such.